### DISCUSSION TITLE
About Char Jump Focus plugin

### DISCUSSION THREAD

**Discussion by fedoranvar** [hyprwm/Hyprland#4597] [Q&A]
Good day to you!

which tool can you recommend for implementing feature, like Jumping (focusing) windows (or workspace) by char (like, `fzf jump`, `tmux select-window`, `
`nvim's plugin lightspeed`, etc.)^

so, i'm `exec` script that showing chars on top of windows, and if i hit a char, focus will be on that window (or workspace)

![image](https://github.com/hyprwm/Hyprland/assets/52028706/ca5a9d4a-333d-4a85-9b22-bcc13eef7cd2)

Thank you!

**Comment by vaxerski** [comment]
if you wanna render over the windows it will have to be a plugin
